examples/meta/generator/translate.py

```diff
@@ -1,17 +1,22 @@
 import json
 import sys
 from string import Template
-from sets import Set
 import os.path
 import argparse
 import re
+# import set in a python 2->3 safe way
```

Comment:
oh this is a patch I recently sent. 
You have to
`git pull --rebase upstream develop`
to avoid this showing up